trial.nf

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env nextflow
+
```

Comment:
You need to enable dsl2 for this to work :smile: 
```suggestion
nextflow.enable.dsl=2
```
Seemed to just work after this change.

Reply:
Ah, I see that this line is technically not needed

Reply:
I'll go ahead and merge